csvkit/join.py

```diff
@@ -1,15 +1,16 @@
 #!/usr/bin/env python
 
 
-def _get_ordered_keys(rows, column_index):
+def _get_keys(rows, column_index, lowercase=False):
     """
-    Get ordered keys from rows, given the key column index.
+    Get keys from rows as keys in a dictionary (i.e. unordered), given the key column index.
     """
```

Comment:
Note that the order of these keys was never used, just `__contains__`, so this actually speeds things up too.